docs/sampleapp.md

```diff
@@ -135,6 +135,17 @@ Without further ado, let's start our application.
   cd ~/orchestration-workshop/dockercoins
   ```
 
+<!--
```

Comment:
Please use single-line style for these <3

Reply:
Oh weird, I thought I had changed them to single-line already. Well, let's change them!